frontend/src/components/Notification/Notification.js

```diff
@@ -1,13 +1,13 @@
 import styles from "./Notification.module.css";
-import { Alert } from "react-bootstrap";
+import React, {useState} from 'react';
 
-let alertType = ""; 
-let alertMainText ="";
-let alertSubText = "";
-let alertHeading = "";
+import {Button} from "react-bootstrap";
+import {Alert} from "react-bootstrap";
+import {Modal} from "react-bootstrap";
 
-function Notification(props) {
+import axios from 'axios';
 
+function Notification(props) {
     [
```

Comment:
Where does this go? A map returns the modified array. It doesn't seem to be used anywhere.